modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -433,7 +433,7 @@ module VnpProcFormCreate
       module VnpProcWebServiceBeanV2
         DEFINITION =
           Bean.new(
-            path: 'VnpProcWebServiceBeanV2',
+            path: 'VnpProcWebServiceBeanV2/VnpProcServiceV2',
```

Comment:
Why are we adding the service to the path here? The other modules only have the Bean in the path. Is this pattern unique to 'Vnp' services?

Reply:
yeah, this is actually causing it to call for the WSDL.